src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1083,6 +1083,15 @@
   <data name="DoNotCreateTasksWithoutPassingATaskSchedulerMessage" xml:space="preserve">
     <value>Do not create tasks without passing a TaskScheduler</value>
   </data>
+  <data name="DoNotAddFinalizerToMemoryManagedTDerivedTypesDescription" xml:space="preserve">
```

Comment:
```suggestion
  <data name="DoNotAddFinalizerToMemoryManagerTDerivedTypesDescription" xml:space="preserve">
```

Reply:
Looks like all of the strings say "Managed" instead of "Manager" in their resource names

Reply:
I'd also drop the "T"

Reply:
OK